ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/retriever/DataColumnSidecarRetriever.java

```diff
@@ -30,6 +30,9 @@ public interface DataColumnSidecarRetriever {
    */
   SafeFuture<DataColumnSidecar> retrieve(DataColumnSlotAndIdentifier columnId);
 
+  /** Starts all the queued retrieve requests immediately */
+  void flush();
```

Comment:
I'm not sure on this way to go. So we just call nextRound on this but it doesn't gurantee that the thing we submitted recently will be executed soon. It could be deep in the queue or whatever.

Reply:
Yeah, probably the javadoc could be more accurate? Or do you have any other approach in mind?

Reply:
Changed the description. Do you think it would work?

Reply:
Looks good!

Reply:
Looks good!